packages/host/app/lib/utils.ts

```diff
@@ -25,3 +25,41 @@ export function stringToColor(string: string | null) {
 
   return color;
 }
+
+export function iconURLFor(word: string) {
+  if (!word) {
+    return undefined;
+  }
+
+  return iconURLs[word.toLocaleLowerCase().charAt(0)];
+}
+
+// TODO move the hosting to our S3
+const iconURLs: { [letter: string]: string } = Object.freeze({
```

Comment:
These icons look like they somehow diverge from the general boxel style, is this intended or is there a plan to replace them?
<img width="383" alt="image" src="https://github.com/user-attachments/assets/8daed9ef-8aec-4df8-a09a-3a0c1a3baa96">

Reply:
These came from @lukemelia, he’ll have to chime in

Reply:
They are a temporary solution made by me. We'll get Gardella to design something more in keeping with the rest of the system.